libs/mpeg-transport-stream/src/test/java/org/codice/ddf/libs/mpeg/transport/MTSValidPacketIteratorTest.java

```diff
@@ -0,0 +1,112 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.libs.mpeg.transport;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.is;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.runners.MockitoJUnitRunner;
+import org.taktik.mpegts.MTSPacket;
+import org.taktik.mpegts.sources.ResettableMTSSource;
+
+/**
+ * Validate the behavior of {@link MTSValidPacketIterator}.
+ */
+@RunWith(MockitoJUnitRunner.class)
+public class MTSValidPacketIteratorTest {
+
+    @Mock
+    private ResettableMTSSource mockSource;
+
+    private List<MTSPacket> mockedValidPackets;
+
+    private Iterator<MTSPacket> validPackets;
+
+    private MTSValidPacketIterator packetIterator;
+
+    @Before
+    public void setup() throws Exception {
+        packetIterator = new MTSValidPacketIterator(mockSource);
+        mockedValidPackets = new ArrayList<>();
+
+        mockedValidPackets.add(mock(MTSPacket.class));
+        mockedValidPackets.add(mock(MTSPacket.class));
+        mockedValidPackets.add(mock(MTSPacket.class));
+
+        validPackets = mockedValidPackets.iterator();
+    }
+
+    @After
+    public void destroy() throws Exception {
+        mockedValidPackets.clear();
```

Comment:
Don't need this since you're assigning `mockedValidPackets` a `new ArrayList<>()` in your `@Before`.